torch/csrc/jit/python/python_ir.cpp

```diff
@@ -526,6 +526,7 @@ void initPythonIRBindings(PyObject* module_) {
           "Find all nodes",
           py::arg("kind"),
           py::arg("recurse") = true)
+      .def("updateLoopInputs", [](Node& n) { return updateLoopNodeInputs(&n); })
```

Comment:
return type of `updateLoopNodeInputs` is void. Is it intended? or should instead return updated n back?